index.d.ts

```diff
@@ -284,7 +268,32 @@ declare module 'plaid' {
     days_available: number;
     historical_balances: Array<HistoricalBalance>;
     transactions: Array<AssetReportTransaction>;
-    owners: Array<Identity>;
+    owners: Array<AssetIdentity>;
+  }
+
+  // Different from "Identity", as it belongs with "Assets"
+  // which is only for US.
+  interface AssetIdentity {
```

Comment:
this is a nit: but can we call this `AssetsIdentity` (and `AssetsAddress` / `AssetsAddressData` ) 

I think the name ideally aligns with the product name.